release/release.go

```diff
@@ -87,6 +88,22 @@ func GenReleaseNotes(ctx context.Context, repo, milestone, prevMilestone, ghToke
 	return nil
 }
 
+// CheckUpstreamRelease takes the given tags and checks
+// for their existence.
+func CheckUpstreamRelease(ctx context.Context, client *github.Client, tags []string) ([]*github.RepositoryRelease, error) {
+	releases := make([]*github.RepositoryRelease, len(tags))
+
+	for _, tag := range tags {
+		release, _, err := client.Repositories.GetReleaseByTag(ctx, "kubernetes", "kubernetes", tag)
```

Comment:
I wonder if we need to hardcode this string over here. What I mean is:

- Is there any value on moving these values either to function scope const or package scope vars (for testing)?
- What using this function for another upstream dependency?

Reply:
I made the function generic and tested. Thank you for this suggestion.